package.json

```
@@ -77,6 +77,7 @@
     "@canonical/cookie-policy": "2.1.0",
     "@canonical/global-nav": "2.4.3",
     "@canonical/latest-news": "1.0.3",
+    "smartquotes": "^2.3.1",
```

```suggestion
    "smartquotes": "^2.3.2",
```

bump version to noscript escape version